### PR TITLE
Report pre-load issues

### PR DESCRIPTION
**Pull request by akurinnoy**
### What does this PR do?

When the Dashboard fails to pre-load some data (plugins/devfiles/workspaces/etc) it warns the user about the issue:

Screenshot 2021-05-13 at 11 00 24

Also the ability to re-enable alerts related to the shown page is added:

https://user-images.githubusercontent.com/16220722/118811815-1de46c80-b8b6-11eb-84fb-96580dad9915.mov

### What issues does this PR fix or reference?

fixes https://github.com/eclipse/che/issues/19564

### CODE REVIEW

**Review by sleshchenko — APPROVED** [review]
Tested with scaling Che Server to 0 and got what you shared in PR description
Screenshot_20210517_141445
Though I haven't review changes

**Comment by akurinnoy** [comment]
@sleshchenko testing PRs is also extremely important

File: src/store/Branding/index.ts [hunk 1]
```diff
@@ -90,14 +100,19 @@ export const reducer: Reducer<State> = (state: State | undefined, incomingAction
...
     case 'REQUEST_BRANDING':
-      return Object.assign({}, state, {
+      return createState(state, {
         isLoading: true,
       });
     case 'RECEIVED_BRANDING':
-      return Object.assign({}, state, {
+      return createState(state, {
         isLoading: false,
         data: action.data,
```
Comment (olexii4) [inline src/store/Branding/index.ts:109]: I have a proposal:
```
    case 'RECEIVED_BRANDING':
      return Object.assign({}, state, {
      return createState(state, {
        isLoading: false,
        data: action.data,
      });
```      
=>
```
    case 'RECEIVED_BRANDING':
      return Object.assign({}, state, {
      return createState(state, {
        isLoading: false,
        data: action.data,
        error: undefined
      });
 ``` 
 
 The same for 
'RECEIVE_REGISTRY_METADATA',
'RECEIVE_NAMESPACES',
'RECEIVE_PLUGINS',
'RECEIVE_DW_PLUGIN',
'RECEIVE_USER',
'RECEIVE_USER_PROFILE',
'RECEIVE_WORKSPACE_SETTINGS',
'CHE_RECEIVE_WORKSPACES'.

WDYT?

Comment (akurinnoy) [reply src/store/Branding/index.ts:109]: Good catch! I've forgotten reset errors in a few stores.

Comment (akurinnoy) [reply src/store/Branding/index.ts:110]: fixed

**Review by olexii4 — APPROVED** [review]
LGTM

**Review by JPinkney — APPROVED** [review]

**Pull request merged**